fdbserver/DDTeamCollection.actor.cpp

```diff
@@ -382,8 +382,16 @@ class DDTeamCollectionImpl {
 			// 	self->traceAllInfo(true);
 			// }
 
-			req.reply.send(std::make_pair(bestOption, foundSrc));
+			if (!self->readyToStart.isReady()) {
```

Comment:
Is this needed? If so, should this be in serverGetTeamRequests?

Reply:
This is needed so that we keep using the original team before TeamCollection is ready. After  TeamCollection is ready, the `trackExcludedServers` actor will mark teams as unhealthy. Then the original unhealthy team will also be returned.

It seems moving to `serverGetTeamRequests` doesn't solve the problem, not sure exactly why.